src/panel/direction/RouteResult.jsx

```diff
@@ -14,6 +14,7 @@ export default class RouteResult extends React.Component {
     vehicle: PropTypes.string,
     isLoading: PropTypes.bool,
     error: PropTypes.bool,
+    errorCode: PropTypes.number,
```

Comment:
I wonder if we couldn't use a single `errorCode` prop in place of `error` *and* `errorCode`. With something like `-1` for unspecified errors and `0` for no error.
A best practice in React is to work with the minimum viable state, to avoid inconsistencies between state values that have some relation.